docs/pages/apis/blaze.mdx

```diff
@@ -5,35 +5,108 @@ import { Callout } from "nextra/components";
 
 Blaze is a NodeJS library for creating Cardano transactions and off-chain code for your Aiken contracts in JavaScript. You can learn more about Blaze [in their documentation site](https://blaze.butane.dev/).
```

Comment:
maybe lets generalize to "Cardano Smart Contracts" instead of Aiken

Reply:
The generalization makes sense, thanks for the review! Will make the change right now